packages/fetch/README.md

```diff
@@ -77,11 +81,11 @@ Use the appropriate method to set the HTTP method being used in the request.
 const fetch = new FetchClient();
 
 fetch.delete('/foo'); // DELETE /foo
-fetch.get('/foo');    // GET    /foo
-fetch.head('/foo');   // HEAD   /foo
-fetch.patch('/foo');  // PATCH  /foo
-fetch.post('/foo');   // POST   /foo
-fetch.put('/foo');    // PUT    /foo
+fetch.get('/foo'); // GET    /foo
```

Comment:
these comments should probably still line up if it's a README.md and intentional formatting

Reply:
I am willing to let prettier win here